atlascope_prototype/client/src/App.vue

```diff
@@ -79,8 +82,8 @@ export default defineComponent({
       })
       if (root._id === parent_id) {
         root.children = children;
-        root.parquet = root.children.find((c) => c.name === root.name + '.parquet');
-        root.image = root.children.find((c) => c.name === root.name + '.svs');
+        root.parquets = root.children.filter((c) => c.name.match('([a-z]+).parquet'));
+        root.image = root.children.find((c) => c.name.match('(.+).svs'));
```

Comment:
1. You probably want `\\.` in place of the `.` here so the dot matches a dot rather than any character.
2. Can both regexes be simplified to a pattern like `'\\.parquet$'` (or even just `.endsWith('.parquet')`)?